package/src/hooks/useLightEntity.ts

```diff
@@ -71,7 +75,24 @@ export function useLightEntity(lightManager: LightManager, config: UseLightEntit
         setIntensity(entity, intensity.value)
       })
     )
-  }, [config.intensity, entity, lightManager, workletContext])
+  })
+
+  // Subscribe to the colorKelvin shared value
+  useWorkletEffect(() => {
+    'worklet'
+    const colorKelvin = config.colorKelvin
+    if (colorKelvin == null) return
+    if (typeof colorKelvin === 'number') return
+
+    const setColor = lightManager.setColor
+
+    return colorKelvin.addListener(
+      workletContext.createRunAsync(() => {
+        'worklet'
+        setColor(entity, makeKelvinLinearRGB(colorKelvin.value))
```

Comment:
I noticed the type def for `linearSRGBColor` being `Float3[]`, shouldn't be just `Float3`? It receives a `[number, number, number]`

Reply:
It's defined like that on its type def on `filament-js` btw

<img width="946" height="159" alt="Image" src="https://github.com/user-attachments/assets/b1576699-8d6d-4ca8-b057-141d99e92557" />

Reply:
Not sure if it's an error, but if it is, I think is not the only place with this problem as well. Please confirm so I can also take a look into that as well.